usr/src/uts/common/fs/zfs/sys/dsl_pool.h

```diff
@@ -103,6 +104,8 @@ typedef struct dsl_pool {
 	kcondvar_t dp_spaceavail_cv;
 	uint64_t dp_dirty_pertxg[TXG_SIZE];
 	uint64_t dp_dirty_total;
+	uint64_t dp_long_free_dirty_pertxg[TXG_SIZE];
+	uint64_t dp_long_freeing_total;
```

Comment:
I think that dp_long_freeing_total is the sum of dp_long_free_dirty_pertxg[].  Why not add up dp_long_free_dirty_pertxg[] in spa_prop_get_config()?